edi_oca/views/edi_exchange_record_views.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="edi_exchange_record_view_tree" model="ir.ui.view">
+        <field name="model">edi.exchange.record</field>
+        <field name="arch" type="xml">
+            <list>
+                <field name="backend_id" />
+                <field name="type_id" />
+                <field name="identifier" />
+                <field name="external_identifier" optional="hide" />
+                <field
+                    name="company_id"
+                    groups="base.group_multi_company"
+                    optional="hide"
+                />
+                <field name="related_name" optional="hide" />
+                <field name="res_id" groups="base.group_no_one" optional="hide" />
+                <field name="model" groups="base.group_no_one" optional="hide" />
+                <field name="exchanged_on" />
+                <field name="ack_received_on" />
+                <field
+                    name="edi_exchange_state"
+                    decoration-success="edi_exchange_state in ['output_sent_and_processed', 'input_processed']"
+                    decoration-danger="edi_exchange_state in ['validate_error', 'output_error_on_send', 'output_sent_and_error', 'input_receive_error', 'input_processed_error']"
+                    widget="badge"
+                />
+            </list>
+        </field>
+    </record>
+    <record id="edi_exchange_record_view_form" model="ir.ui.view">
+        <field name="model">edi.exchange.record</field>
+        <field name="arch" type="xml">
+            <form string="EDI Exchange Record">
+                <field name="direction" invisible="1" />
+                <field name="retryable" invisible="1" />
+                <field name="related_record_exists" invisible="1" />
```

Comment:
FYI, required field (invisible) added automatically in views. (See `https://github.com/odoo/odoo/pull/137031`)

Other similar places

```suggestion

```

Reply:
@thienvh332 done :)